catalog/imaging/catalog-imaging-transformer/src/main/java/com/connexta/alliance/transformer/nitf/NitfPreStoragePlugin.java

```diff
@@ -95,12 +95,21 @@ private void process(List<ContentItem> contentItems) {
         ContentItem contentItem = contentItems.get(0);
         Metacard metacard = contentItem.getMetacard();
 
+        if (contentItem.getMimeTypeRawData() == null || !contentItem.getMimeTypeRawData()
```

Comment:
Could simplify this a bit:
`if (!NitfInputTransformer.MIME_TYPE.equals(contentItem.getMimeTypeRawData()))`